src/css/skins/five.less

```diff
@@ -133,4 +131,24 @@
         border-radius: @five-border-radius;
     }
 
+    // override control bar element colors that don't look good in time slider above mode
+    &.jw-flag-time-slider-above,
+    &.jw-breakpoint-0,
+    &.jw-breakpoint-1 {
+      .jw-controlbar {
+        .jw-text {
```

Comment:
@egreaves The goal here was to only override those colors for control bar elements and when in time slider above flag/small player size. We don't want to override the colors anywhere else. However, there is a cleaner way I could do this.. thinking about it.